app/RouteHandlers/AddDonationHandler.php

```diff
@@ -93,6 +93,15 @@ private function newHttpResponse( AddDonationResponse $responseModel ): Response
 					)
 				);
 				break;
+			case PaymentType::SOFORT:
+				$httpResponse = $this->app->redirect(
+					$this->ffFactory->newSofortUrlGeneratorForDonations()->generateUrl(
```

Comment:
We _do_ have global exception handling. What would be your suggestion how we handle said exception? We fail to produce an URL we wanted to use to redirect her/him, would need an alternative user flow. It's really a more a question of UX than technicality.